extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+
+#### Projection
+
+| Field Name                            | Description                                                  | Req.      |
+| ------------------------------------- | ------------------------------------------------------------ | --------- |
+| proj:epsg / proj:wkt2 / proj:projjson | **REQUIRED.** At least one of the three properties `proj:epsg`, `proj:wkt2` or `proj:projjson` is required. It MUST specify the coordinate reference system (1.5) and MAY specify the map projection (1.6). | 1.5 / 1.6 |
+
+#### View
+
+| Field Name           | Description                                                  | Req. |
+| -------------------- | ------------------------------------------------------------ | ---- |
+| view:off_nadir       | **REQUIRED.** The average off-nadir angle, for per-pixel angles, refer to the asset with the key `off-nadir`. Convert to degree, if required. | \*   |
+| view:incidence_angle | **REQUIRED.** The average incidence angle, for per-pixel angles, refer to the asset with the key `incidence-angle`. Convert to degree, if required. | \*   |
+| view:azimuth         | **REQUIRED.** The average azimuth angle, for per-pixel angles, refer to the asset with the key `azimuth`. Convert to degree, if required. | \*   |
+| view:sun_azimuth     | **REQUIRED.** The average sun azimuth angle, for per-pixel angles, refer to the asset with the key `sun-azimuth`. Convert to degree, if required. | \*   |
+| view:sun_elevation   | **REQUIRED.** The average sub elevation angle, for per-pixel angles, refer to the asset with the key `sun-elevation`. Convert to degree, if required. | \*   |
+
+\* = Requirement 2.8 for Surface Temperature (ST) / 2.11 for Surface Reflectance (SR)
+
+### STAC Item Links
+
+| Relation Type            | Description                                                  | Req.                |
+| ------------------------ | ------------------------------------------------------------ | ------------------- |
+| card4l-document          | **REQUIRED.** Provides at least one link to the CARD4L specification document. Word (media type: `application/vnd.openxmlformats-officedocument.wordprocessingml.document`) and/or PDF (media type: `application/pdf`). | *n/a*               |
+| derived_from             | Points back to the source's STAC Item. May be multiple items, if the product is derived from multiple acquisitions. | 1.15                |
+| about                    | Link to algorithms used in the generation process. See also the [notes](#notes) regarding req. 1.13. | 1.13                |
+| related                  | **REQUIRED.** Link to the sources of ancillary or auxiliary data used in the generation process. Excludes DEMs, which use the relation `elevation-model` instead. | 1.14                |
+| access                   | STRONGLY RECOMMENDED. Link to data access information.       | 1.16                |
```

Comment:
So, a landing page to this dataset?
Seems more useful at the Collection level

Reply:
There's no req for having a Collection

Reply:
I've added the following chapter to the spec, which should help users to de-duplicate assets and links:

## STAC Collections

CARD4L lists a lot of requirements (and fields) that have common values across all generated STAC Items and assets.
Thus, it is **recommended** to provide a STAC Collection for the Items and put common fields (in the STAC Item `properties`)
into [Collection `summaries`](../../collection-spec/collection-spec.md#collection-fields).
While the STAC Item fields still need to be in the Item, too, you can de-duplicate links and assets by putting common
links once into the STAC Collection links. Also, common assets can be just put once into the STAC Collection using the
STAC extension [Collection Assets](../collection-assets/README.md).